pkg/model/bootstrapscript_test.go

```diff
@@ -57,20 +58,53 @@ func TestBootstrapUserData(t *testing.T) {
 	cs := []struct {
 		Role             kops.InstanceGroupRole
 		ExpectedFilePath string
+		HookSpecRoles    []kops.InstanceGroupRole
 	}{
 		{
 			Role:             "Master",
 			ExpectedFilePath: "tests/data/bootstrapscript_0.txt",
```

Comment:
Total nit pick question.  Why do we need 5 test files? A quick eye ball was not showing me any differences.